package.json

```diff
@@ -13,8 +13,9 @@
     "build.test": "gulp build.test --color",
     "build.test.watch": "gulp build.test.watch --color",
     "generate.manifest": "gulp generate.manifest --color",
-    "e2e": "protractor",
-    "e2e.live": "protractor --elementExplorer",
+    "e2e": "protractor src/browser/protractor.conf.js",
+    "e2e.live": "protractor src/browser/protractor.conf.js --elementExplorer",
+    "e2e.singleRun": "gulp build.prod --color && gulp build.js.e2e --color && gulp e2e --color",
     "gulp": "gulp",
     "karma": "karma",
```

Comment:
Hey @Shyam-Chen, what do you mean?

About `"e2e"` -> I had to add the `src/browser/protractror.conf.js` to it, since it has been moved to `src/browser`. The same for `"e2e.live"`. 

About `"e2e.singleRun"` i renamed `"e2e.cli"` as you suggested to `"e2e.singleRun"` and sorted it alphabetically under `"e2e.live"`(since all other scripts are also in alphabetical order)